chatkit-android/build.gradle

```diff
@@ -66,6 +66,13 @@ dependencies {
     androidTestImplementation("com.android.support.test.espresso:espresso-core:$espresso_version", {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
+
+    api 'com.github.pusher:push-notifications-android:add-users-support-SNAPSHOT'
```

Comment:
Is this publicly published? As in, if we release with this dependency, will people be able to build?

Reply:
So jitpack.io allows you to get builds from branches. We were planning to create a branch called `chatkit-release-1.0.0` so that we don't need to make an official (maven) release for the Beams Push Notifications SDK. Our users wouldn't just yet get this new feature, but Chatkit users would. Alternatively, we can publish to maven a version under a different name.

This is a temporary solution until we release the Beams SDKs official Users support and move Chatkit unto the mainline.

Reply:
We now have dedicated ck release, so we can change this :)